src/script/hydra-create-user

```diff
@@ -129,6 +141,24 @@ $db->txn_do(sub {
         if (defined $passwordHash) {
```

Comment:
Should the `!(defined $passwordHash)` just above this be changed to also take into account `$passwordPrompt`?

Or maybe that conditional shoud be removed altogether? We already `die` if more than one of `$passwordHash`, `$password`, and `$passwordPrompt` are defined around line 95...

Reply:
Good catch, we can delete that extra condition.